enterprise/server/cmd/ci_runner/main.go

```diff
@@ -717,9 +717,13 @@ func run() error {
 		if err != nil {
 			log.Warningf("Failed to read BuildBuddy config; will run `bazel shutdown` from repo root: %s", err)
 		}
-		wsPath := ""
+		cwd, err := os.Getwd()
```

Comment:
We do a lot of changing directories, so this logic feels a bit brittle. Could we reuse this logic here `bazelWorkspaceAbsPath := filepath.Join(ws.rootDir, repoDirName, action.BazelWorkspaceDir)` and consolidate it in `bazelArgsWithCustomBazelrc`?

Reply:
Done